language/move-lang/tests/move_check/parser/let_binding.move

```diff
@@ -0,0 +1,29 @@
+module M {
+    struct R {
+        f: u64
+    }
+    struct Generic<T> {
+        g: T
+    }
+    f() {
+        let () = ();
+        let (): () = ();
+        // FIXME: The grammar currently treats "()" as a single token, but it is
+        // not clear if that has any advantage compared to separate tokens that
+        // allow whitespace in between them.
+        // let ( ) = ( );
+        // let ( ): ( ) = ( );
+        let v1 = 1;
+        let v2: u64 = 2;
+        // FIXME: The grammar does not currently accept a single variable inside
+        // parens, but for consistency, maybe that ought to be allowed.
+        // let (v3) = 3;
```

Comment:
We might want to allow it, yeah. I am undecided

Reply:
The new parser already allows this, so I went ahead and uncommented the test. If you decide you really don't like it, I'm certainly open to discussing it.